### PR TITLE
Quickstart

### PR DESCRIPTION
**Pull request by wtrocki**
## Motivation

Provide essence of the RainCatcher to guide developer in 10 minutes thru all functionalities.

Tried 2 options
- Include existing content in quickstart
- Duplicate content as bellow.

I personally thing that none of them work.
Duplication seems to be not an option as I will need to include exactly the same content. 
Including it's duplicating content in the overall documentation which is little bit confusing for people where they should get that. 

We have now getting started and step documentation that covers that case.
I wanted to get your opinions.

### CODE REVIEW

File: docs/workforce-management-framework/topics/ref-quickstart.adoc [hunk 1]
```diff
@@ -0,0 +1,44 @@
+[id='{context}-ref-raincatcher-quickstart']
+= RainCatcher QuickStart
+
+== Requirements
+
+- Node.js
+- MongoDB
+- Redis
+
```
Comment (witmicko) [inline docs/workforce-management-framework/topics/ref-quickstart.adoc:9]: maybe add recommended versions?

**Review by witmicko — APPROVED** [review]
lgtm, one comment

**Comment by wtrocki** [comment]
@witmicko see my comment above.

**Comment by witmicko** [comment]
@wtrocki how about we'd create raincatcher-gettingstarted repo and just have this PR's content in the README? we could then link to that from other readmes
I agree that duplication is bad, but at the moment the information is spread out a bit too much, collated version like this will help to really get started in 10m

**Comment by wtrocki** [comment]
@witmicko Maybe we can just have video as part of that section. 
I checked that already and we can just put youtube video directly.

I'm ok to merge current content but it feels little bit incomplete. 
Repo seems to be additional hassle and another thing to update/document. 
Less artifacts makes it east to maintain.

File: docs/workforce-management-framework/topics/ref-quickstart.adoc [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+----
+
+4. Create step
+
+[source,bash]
+----
+yo rcstep
+---
+
+
```
Comment (JameelB) [inline docs/workforce-management-framework/topics/ref-quickstart.adoc:43]: Missing start apps step?

File: docs/workforce-management-framework/topics/ref-quickstart.adoc [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+----
+npm install -g yo
+npm install -g generator-rcstep
+----
+
+4. Create step
+
+[source,bash]
+----
+yo rcstep
```
Comment (JameelB) [inline docs/workforce-management-framework/topics/ref-quickstart.adoc:40]: where is this supposed to be ran? Is this step valid for all the apps?

**Review by JameelB — COMMENTED** [review]
Looks fine to me so far, just one small comment. Since it is a quickstart, the less background info the better and just focus on getting the apps running. If they wanted to find more info then they'll just have to read on further in the docs. I think having the video as part of this section would be a good idea.

**Comment by wtrocki** [comment]
I'm going to close that as this may not have sense. Done some improvements and hosting video as part of documentation and website.

**Pull request closed without merging**